openstack/02a-MAClow/IEEE802154E.c

```diff
@@ -927,7 +931,7 @@ port_INLINE void activity_ti1ORri1() {
          // assuming that there is nothing to send
          ieee154e_vars.dataToSend = NULL;
          // check whether we can send
-         if (schedule_getOkToSend()) {
+         if (schedule_getOkToSend() || cellType==CELLTYPE_TX) {

```

Comment:
you are bypassing the `schedule_getOkToSend()` function by putting the `||` statement!